src/core/alloc.h

```diff
@@ -26,7 +38,11 @@ MVM_STATIC_INLINE void * MVM_realloc(void *p, size_t size) {
 }
 
 MVM_STATIC_INLINE void * MVM_recalloc(void *p, size_t old_size, size_t size) {
+#ifdef MVM_USE_MIMALLOC
+    void *ptr = mi_realloc(p, size);
```

Comment:
There is an `mi_recalloc` https://github.com/microsoft/mimalloc/blob/master/include/mimalloc.h#L227, but it expects a new count and size (like regular calloc), not old size and new size. There are only a couple uses of `MVM_recalloc`, maybe a followup commit/PR could switch `MVM_recalloc` to use `mi_recalloc` and fix all the uses.

Reply:
https://gist.github.com/MasterDuke17/24f85769c96d78166138d0c18c792020 has a diff against master that I believe changes all the uses of `MVM_recalloc` and just needs a tweak in the implementation of `MVM_recalloc` to use `mi_recalloc`.